src/main/scala/stellar/sdk/inet/HorizonAccess.scala

```diff
@@ -69,9 +69,19 @@ class Horizon(uri: URI,
     }
   }
 
-  def getStream[T: ClassTag](path: String, de: CustomSerializer[T], params: Map[String, String] = Map.empty)
+  def getStream[T: ClassTag](path: String, de: CustomSerializer[T], cursor: HorizonCursor, order: HorizonOrder, params: Map[String, String] = Map.empty)
                             (implicit ec: ExecutionContext, m: Manifest[T]): Future[Stream[T]] = {
 
+    val orderParam = order match {
+      case Asc => "asc"
+      case Desc => "desc"
+    }
+    val cursorParam = cursor match {
+      case Now => "now"
+      case Record(_) => cursor.value.toString
```

Comment:
If `HorizonCursor` is kept, this can be `case Record(r) => r.toString`.
If simplified to `Option[Long]` (see below), this can be `cursorParam.map(_.toString).getOrElse("now")`